edu-services/gradebook-service/hibernate/src/java/org/sakaiproject/tool/gradebook/CourseGradeRecord.java

```diff
@@ -216,6 +216,12 @@ public void setCalculatedPointsEarned(final double literalTotalPointsEarned) {
 		this.calculatedPointsEarned = literalTotalPointsEarned;
 	}
 
+	// Added compatibility in GB Classic (12.x) - SAK-41295
+	public void initNonpersistentFields(final double totalPointsPossible, final double totalPointsEarned,
+			final double literalTotalPointsEarned) {
```

Comment:
Don't finalise the method params. You just don't need to. I know the rest of the methods use final all the time, but it's really not needed when they're finalised in the following call. final is just overused, imo, and it clutters the method declaration. I love final when you declare a var early on in a long method, or if you're parallellising some code, but otherwise - they're just noise.

Reply:
Deleted final on params. Thanks @adrianfish!